cylc/flow/platforms.py

```diff
@@ -190,10 +209,6 @@ def platform_from_name(
     if platform_name is None:
         platform_name = 'localhost'
 
-    platform_group = None
-    # The list is reversed to allow user-set platform groups (which are loaded
-    # later than site set platforms) to be matched first and override site
-    # defined platforms.
```

Comment:
Useful comment deleted?

Reply:
Useful comment moved to function docstring. I'm open to moving it back.

Reply:
Imho it's better next to the line that is in question